aviary/subsystems/propulsion/engine_sizing.py

```diff
@@ -21,12 +21,9 @@ def initialize(self):
             desc='collection of Aircraft/Mission specific options')
 
     def setup(self):
-        # count = len(self.options['aviary_options'].get_val('engine_models'))
-        count = 1
-
-        add_aviary_input(self, Aircraft.Engine.SCALED_SLS_THRUST, val=np.zeros(count))
+        add_aviary_input(self, Aircraft.Engine.SCALED_SLS_THRUST, val=0.0)
 
-        add_aviary_output(self, Aircraft.Engine.SCALE_FACTOR, val=np.zeros(count))
+        add_aviary_output(self, Aircraft.Engine.SCALE_FACTOR, val=0.0)
```

Comment:
this component is just for enginedecks, because not all engines use it we can't do all engines at once. A separate copy of this component is made for each engine that needs it

Reply:
ahh, thanks